src/components/metadata/MetaObject.js

```diff
@@ -21,6 +21,7 @@ export class MetaObject extends Component {
       let type = 'simple';
       if (_.isObject(value)) type = 'object';
       if (_.isArray(value)) type = 'array';
+      if (_.isArray(value) && key === 'tags') type = 'simple';
```

Comment:
Could you extract these conditions into a method which returns type given value?

Reply:
Good idea. Will handle this in a separate PR..